scripts/pull_github_pr.sh

```diff
@@ -36,6 +48,52 @@ curl() {
     command curl "${opts[@]}" "$@"
 }
 
+set_jenkins_job() {
+  branch=$(git rev-parse --abbrev-ref HEAD)
+  product=$(cat SCYLLA-VERSION-GEN | grep ^PRODUCT | awk -F'=' '{print $2}')
+
+
+  if [[ "$branch" == "next" ]]; then
+    jenkins_job="scylla-master/job/next"
+  fi
+  if [[ "$branch" == "next-enterprise" ]]; then
+    jenkins_job="scylla-enterprise/job/next"
+  fi
+  if [[ "$branch" == next-* ]]; then
+    version="${branch#next-}"
+    folder_prefix="scylla"
+    if [[ "$product" == "scylla-enterprise" ]]; then
+      folder_prefix="enterprise"
+    fi
+    jenkins_job="$folder_prefix-$version/job/next"
+  fi
+}
+
+check_jenkins_job_status() {
+  set_jenkins_job
+
+  lastCompletedJobName="$jenkins_url/job/$jenkins_job/lastCompletedBuild"
+  lastCompletedBuild="$lastCompletedJobName/api/json?tree=result"
+  getBuildResult=$(curl -s --user $JENKINS_USERNAME:$JENKINS_API_TOKEN $lastCompletedBuild/api/json?tree=result)
+  if [[ "$getBuildResult" == *"Unauthorized"* ]]; then
```

Comment:
Please put the "*" inside the quotes, not outside. It looks like filename globbing this way (even if the bash [[ avoids the globbing, it still looks very weird).

Reply:
Fixed